x/oracle/types/events.go

```diff
@@ -15,19 +15,20 @@ const (
 	EventTypeRemoveReporter     = "remove_reporter"
 	EventTypeResolve            = "resolve"
 
-	AttributeKeyID             = "id"
-	AttributeKeyDataSourceID   = "data_source_id"
-	AttributeKeyOracleScriptID = "oracle_script_id"
-	AttributeKeyExternalID     = "external_id"
-	AttributeKeyDataSourceHash = "data_source_hash"
-	AttributeKeyCalldata       = "calldata"
-	AttributeKeyValidator      = "validator"
-	AttributeKeyReporter       = "reporter"
-	AttributeKeyClientID       = "client_id"
-	AttributeKeyAskCount       = "ask_count"
-	AttributeKeyMinCount       = "min_count"
-	AttributeKeyResolveStatus  = "resolve_status"
-	AttributeKeyGasUsed        = "gas_used"
-	AttributeKeyResult         = "result"
-	AttributeKeyReason         = "reason"
+	AttributeKeyID                     = "id"
+	AttributeKeyDataSourceID           = "data_source_id"
+	AttributeKeyOracleScriptID         = "oracle_script_id"
+	AttributeKeyExternalID             = "external_id"
+	AttributeKeyDataSourceHash         = "data_source_hash"
+	AttributeKeyCalldata               = "calldata"
+	AttributeKeyValidator              = "validator"
+	AttributeKeyReporter               = "reporter"
+	AttributeKeyClientID               = "client_id"
+	AttributeKeyAskCount               = "ask_count"
+	AttributeKeyMinCount               = "min_count"
+	AttributeKeyResolveStatus          = "resolve_status"
+	AttributeKeyGasUsed                = "gas_used"
+	AttributeKeyDataSourceFeeCollected = "data_source_fee_collected"
```

Comment:
AttributeKeyTotalFees= "total_fees" -> for request event
AttributeKeyFee= "fee" -> for raw request event

Reply:
Fixed